roles/node/tasks/main.yml

```diff
@@ -4,6 +4,10 @@
 - name: Install AvalancheGo
   include_tasks: install-avalanchego.yml
 
+- name: Clean plugins dir
```

Comment:
What happens if we only remove 1 VM? Shouldn't we loop over the VMs and remove each one that is not listed?

Reply:
Refactored in that sense